use-case-guides/workday-request-time-off/README.md

```diff
@@ -1,131 +1,104 @@
-
-By providing a direct and intuitive way for employees to submit time off requests through their Moveworks bot, businesses remove significant barriers to submitting these requests. This user-friendly interaction not only enhances the employee experience but also encourages adherence to formal request procedures. Employees can quickly navigate through the time-off request process without the need to log into multiple systems or navigate complex menus, making it more likely they will follow company policy and timelines for requests. 
+Your employees deserve to take time off to recharge and stay productive. Requesting time off in Workday requires you to log in and navigate multiple screens.
 
+This guide helps you build a conversational experience for easy time off requests for your employees. It ensures your managers and payroll teams have accurate records of time off requests.
 
 Let's dive in!
 
-### **System Overview**
-
-**Depends on**: `Workday connector`([link](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday))
-
-**Pre-requisites**:
-
-1. You have synchronized your employees’ work email from your IDAM like `Okta` or `Microsoft Entra` to Moveworks using Moveworks Setup ([link](https://help.moveworks.com/docs/ingest-users))
-2. Your workday instance uses the same work email as your IDAM.
-
-**Ease of Building**:
-
-This is a `intermediate` difficulty use case
-
-This will take `2 weeks` for your developer and the business system manager to build, test, and launch.
+**Prerequisites**:
 
-**Business Value**:
+1. Synchronize your employees’ work email from your IDAM, such as Okta or Microsoft Entra, to Moveworks [using Moveworks Setup](https://help.moveworks.com/docs/ingest-users).
+2. Ensure that your Workday instance uses the same work email as your IDAM.
+3. Ensure that your Workday administrator has the necessary permissions to configure an API Client for Workday, and provide access to the `Worker Data` and `Time Off` Business Objects as part of domain security policies.
+4. Click on 'Experimental Features' in the top right corner of your Creator Studio window, and ensure that Paths in Copilot and the Next-gen API editor are enabled.
 
-1. Employees need to file time off requests promptly for accurate record keeping
-2. However, it is painful for employees to signing into your single-sign on provider, signing into workday, navigating to your employee schedule calendar, and selecting the appropriate time off plans and dates
-3. If managers don’t have accurate record of when employees are off, they can’t manage
-4. If payroll team does not have accurate records, they will not pay employees properly for time off.
-5. This plugin provides a convenient way for your employees to submit time off requests.
+    ![Experimental Features](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-alt-1.png)
```

Comment:
Can you link me to where in your style guide you're highlighting how experimental feature references will be documented. I think i gave this as verbal feedback last time to not include this.

Reply:
addressed with generic text